src/scanner.c

```diff
@@ -388,30 +449,53 @@ bool tree_sitter_scala_external_scanner_scan(void *payload, TSLexer *lexer,
     skip(lexer);
   }
 
-  if (valid_symbols[SIMPLE_STRING] && lexer->lookahead == '"') {
+  if (valid_symbols[SIMPLE_STRING_START] && lexer->lookahead == '"') {
```

Comment:
Is there a reason we have to scan simple string starts externally instead of just having a normal token, like we do for interpolated strings?

Reply:
It looks like this has been around for a long time, and was originally added for interpolated string too https://github.com/tree-sitter/tree-sitter-scala/commit/2dcb6b04b2123a8c9cce2ca0cd43cc40444530e8. It's possible that it was implemented that way back then, but we could move `"` and `"""` into the parser?